clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/api/KubernetesClientApiAdapter.groovy

```diff
@@ -204,6 +213,22 @@ class KubernetesClientApiAdapter {
     }
   }
 
+  boolean hardDestroyStatefulSet(String name, String namespace) {
+    exceptionWrapper("statefulSets.delete", "Hard Destroy Stateful Set $name", namespace) {
+      V1DeleteOptions deleteOptions = new V1DeleteOptions()
+      Boolean orphanDependents = true
```

Comment:
Are you sure you want to orphan dependents? I don't think that qualifies as a `hardDestroy` since you'll be leaving pods around.

Reply:
The pods will be removed through resize.  I an change the name because resize is not part of this logic.